### DISCUSSION TITLE
Are there plans to support hot updates?

### DISCUSSION THREAD

**Discussion by 0x1af2aec8f957** [tauri-apps/tauri#14548] [Plugin Requests]
### Describe the problem

Hot updates are a very common way to update apps on mobile devices. It is often unwise to ask users to re-download the entire app to fix a minor issue. Hot updates have many advantages over [full updates](https://v2.tauri.app/plugin/updater).

### Describe the solution you'd like

This functionality is similar to [eas-update](https://docs.expo.dev/eas-update/introduction) or [code-push](https://learn.microsoft.com/en-us/appcenter/distribution/codepush).

### Additional context

Almost all technology stacks that use `js|html|css` for application development on mobile devices support this function.

**Comment by FabianLars** [comment]
Tauri's partner CrabNebula has a paid solution for web assets updates ("paid" because it's bound to their Cloud offering) -> https://docs.crabnebula.dev/plugins/over-the-air-updates/
